db/migrate/20220223215832_add_enabled_to_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddEnabledToItems < ActiveRecord::Migration[5.2]
+  def change
+    add_column :items, :enabled, :boolean, :default => true
```

Comment:
nice job updating the migrations

Reply:
If everyone can pull this branch to their local first before we merge, it would be a good idea. We got this to work and didnt need to change any code but there was some issues with the database. We did a few things but it seemed to work mostly after we just restarted a new terminal session and ran db:{drop,create,migrate}